backend/controller/admin/admin.go

```diff
@@ -30,6 +31,15 @@ type SchemaRetriever interface {
 	GetActiveSchema(ctx context.Context) (*schema.Schema, error)
 }
 
+type streamSchemaRetriever struct {
```

Comment:
Why not use the event source directly?

Reply:
There is some code that retrieves schema from disk, I'm not sure how important it is.

Reply:
https://github.com/TBD54566975/ftl/blob/5a999165914490d8ed0f095343f518132c8a5bae/backend/controller/admin/local_client.go#L27